src/pages/Login/Login.scss

```
@@ -0,0 +1,97 @@
+.Login {
+  a {
+    text-decoration: none;
+  }
+  font-family: 'NotoSans';
+  max-width: 1200px;
+  width: 1200px;
+  height: 300px;
+  margin: 0 auto;
+  text-align: center;
+
+  .title {
+    font-size: 24px;
+    font-weight: 500;
+    margin-top: 70px;
+    margin-bottom: 15px;
+  }
+
+  .loginBox {
+    margin: 0 auto;
+    width: 300px;
+
+    .inputText {
+      width: 100%;
+      height: 50px;
+      display: block;
+      font-size: 16px;
+      font-weight: 500;
+      margin-bottom: 12px;
+      outline: unset;
+      border: none;
+      border-bottom: 1px solid #c0c0c0;
+    }
+
+    .loginBtn {
+      width: 100%;
+      height: 55px;
+      background-color: #333333;
+      border: none;
+      color: white;
+      font-size: 18px;
+      font-weight: 800;
+      cursor: pointer;
+    }
+    .loginBtn:hover {
+      background-color: #000000;
+    }
```

부모 선택자 사용해서 다음처럼 작성하실 수 있습니다.
```suggestion
      &:hover {
        background-color: #000000;
      }
    }
```

---

그리고 html요소의 순서에 맞게 css 작성해주시는 게 가독성 및 유지보수에 도움이 됩니다